cloud-function/src/app.ts

```diff
@@ -60,6 +60,13 @@ router.get(
   resolveIndexHTML,
   proxyContent
 );
+router.get(
+  "/[^/]+/blog/*",
+  requireOrigin(Origin.main),
+  redirectLocale,
+  resolveIndexHTML,
+  proxyContent
+);
```

Comment:
🟠 Isn't this already handled by the (last) fallback handler (which also applies to `/plus`), or does this not work because of the trailing slash?

Reply:
Yes, trailing slash is the issue here.